<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vAgregarTrenes</class>
 <widget class="QDialog" name="vAgregarTrenes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Tren</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar Tren</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>181</width>
     <height>99</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtfCapitan"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Placa</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtfPlaca"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Destino</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cmbxDestino">
      <item>
       <property name="text">
        <string>Guanacaste</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Heredia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alajuela</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>San José</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cartago</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Puntarenas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Limón</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Capitán</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
